webviz_ert/assets/webviz-config.yml

```diff
@@ -13,9 +13,7 @@ pages:
  - title: Parameter Comparison Viewer
    content:
     - ParameterComparison:
-   experimental: True
```

Comment:
This works :+1:  but I was thinking of just replacing `experimental: True` with `experimental: False` so that we know there is functionality for setting a page experimental and it is optional and can be activated by setting the flag to `True`

Reply:
I see, in this case, for consistency, I would add the flag `experimental: False` to all of them. Is it ok?

Reply:
Actually, no the point is that this flag is optional if you add it for all the pages then if somebody new reads the config file in a few months will think that they need to set the flag for a new page as well which is not true.

Reply:
I see the point. I think it makes sense since we want to add the flag `beta` on the experimental views, if I understand correctly.